src/oatpp-swagger/oas3/Generator.cpp

```diff
@@ -124,10 +127,12 @@ oatpp::Object<Schema> Generator::generateSchemaForTypeObject(const Type* type, b
     result->type = "object";
     result->properties = {};
 
+    auto instance = type->creator().get();
```

Comment:
You have a memory leak here.
A call to `creator()` returns `oatpp::Void` which you can treat as a smart-pointer. Since you are not using 
smart-pointer but a direct pointer, the object gets destroyed immediately after the call.

The correct code for this block would be:

```cpp
auto instance = type->creator(); // use oatpp::Void smart-pointer. Not void* directly.
auto properties = type->propertiesGetter();

for(auto* p : properties->getList()) {
  const auto& defaultValue = p->get(instance.get()); // use oatpp::Void smart-pointer
  result->properties[p->name] = generateSchemaForType(p->type, true, usedTypes, p, defaultValue);
}
```

Reply:
Oh, thanks! Will try that!